### PR TITLE
Announcing OpenSearchCon North America 2024 Blog

### LINKED ISSUE #2896: OpenSearchCon North America 2024 Registration and CFP are Open
**Describe the blog post your would like to write**
Provides an update to the community re OpenSearchCon North America 2024, including its location, registration info, and how to submit your CfP. 

**What is the title of the blog post?**
OpenSearchCon North America 2024 Registration and CFP are Open

**Who are the authors?**
Daryll Swager

**What is the proposed posting date?**
5/23

### PR DESCRIPTION
**Pull request by nateynateynate**
### Description

Publishes a blog announcing the CFP and registration for OpenSearchCon North America 2024. 
 
### Issues Resolved
Resolves #2896 

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+layout: post
+title:  "Announcing OpenSearchCon North America 2024"
+authors:
+  - daryllswager
+date:   2024-06-01 0:00:10 -0700
+categories:
+  - community
+excerpt: The OpenSearch Project invites the OpenSearch community to explore the future of search, analytics, and generative AI at the first OpenSearch user conference in Europe. Join us in Berlin on May 6 & 7 and learn how to build powerful applications and get the most out of your OpenSearch deployments.
+meta_keywords: opensearchcon Europe, opensearchcon eu, opensearchcon emea, opensearchcon may, opensearchcon call for papers, opensearchcon call for presentations, opensearchcon registration, register for opensearchcon
+meta_description: OpenSearchCon Europe 2024 is coming to Café Moskau in Berlin May 6 and 7. Register today to save your seat!
```
Comment (pajuric) [inline _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md:11]: Updated meta: 
**_meta_keywords:_**  opensearchcon north america, opensearchcon na, opesearchcon call for papers, register for opensearchcon, opensearch community

**_meta_description:_** Join the OpenSearch Project in San Francisco for it’s third annual OpenSearchCon North America 2024 taking place September 24-26 at the Hilton Union Square. Register today.

Comment (nateynateynate) [reply _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md:11]: All set thanks Patti.

Comment (natebower) [reply _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md:11]: I'm assuming this text should reference the North American conference.

File: _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+The [Call for Presentations](https://opensearch.org/events/opensearchcon/2024/north-america/cfp.html) is open until the end of the day on June 14. If you have a story you’d like to share, submit your talk NOW - time is of the essence!
+
+Builders will be presenting dozens of sessions across four conference tracks to help you learn more about building applications at scale as an open-source contributor.  The four conference tracks are:
+
+* Search - for all types of search technology enthusiasts
+* Analytics, Security, Observability - for those that like to visualize and make data-driven decisions
+* Operating OpenSearch - for talks about performance, scalability, interesting implementations, and the like
+* Community - focuses on how communities form and grow in the open-source universe, including OpenSearch
```
Comment (natebower) [inline _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md:48]:
```suggestion
* Community: Focuses on how communities form and grow in the open-source universe, including OpenSearch
```

File: _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+Given the popularity of last year’s event, our partner round table will return this year. This forum allows partners to talk about the project with the OpenSearch team and each other. The conversation spans a wide variety of topics both highly technical and more community-oriented, and usually focuses on the things that are going really well in the project, and areas that could use a little help.  This is a great forum to listen to feedback from experienced partners who have a vested interest and  opinions about the OpenSearch Project.
```
Comment (natebower) [inline _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md:53]:
```suggestion
Given the popularity of last year’s event, our partner roundtable will return this year. This forum allows partners to talk about the project with the OpenSearch community. The conversation usually focuses on things that are going really well with the project and areas that could use improvement. This is a great opportunity to listen to feedback from experienced partners who have a vested interest in the OpenSearch Project.
```

File: _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+The Unconference will be back as well! First introduced last year, this session format allows you to propose a short talk. Your peers will then vote on whether they want to hear from you and a handful of short sessions will be delivered. These sessions have proven to be fun as well as thought-provoking, so don’t miss out.
```
Comment (natebower) [inline _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md:55]:
```suggestion
The Unconference will be back as well! First introduced last year, this session format allows you to propose a short talk. Your peers will then vote on whether they want to hear from you, and a handful of short sessions will be delivered. These sessions have proven to be fun as well as thought provoking, so don’t miss out.
```

Comment (natebower) [inline _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md:55]: `Unconference` => `"unconference"`?

File: _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+On September 26, we’re going to have smaller format training sessions and workshops with enough room for 100 or so people who want to get a more in-depth look at OpenSearch from some of the experts in the field.  Separate signups will be provided for that once the schedule has been finalized, so remember to visit the conference site often to see the latest news and session information.
```
Comment (natebower) [inline _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md:57]:
```suggestion
On September 26, we'll be offering smaller training sessions and workshops with enough room for around 100 people who want to get a more in-depth look at OpenSearch from experts in the field. Separate sign-ups will be provided once the schedule has been finalized, so remember to visit the conference site often for the latest news and session information.
```

File: _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+First and foremost, we just want everyone to come out, have some fun, and learn a bunch of new stuff along the way. See you in San Francisco!
```
Comment (natebower) [inline _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md:59]:
```suggestion
Above all, we just want everyone to come out, have some fun, and learn a lot along the way. See you in San Francisco!
```

File: _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+**Resources:**
```
Comment (natebower) [inline _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md:61]:
```suggestion
### Resources
```

File: _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+[Registration Page](https://opensearch.org/events/opensearchcon/2024/north-america/register.html)
```
Comment (natebower) [inline _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md:63]:
```suggestion
[OpenSearchCon 2024: North America Registration](https://opensearch.org/events/opensearchcon/2024/north-america/register.html)
```

File: _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+[CFP Submission Page](https://opensearch.org/events/opensearchcon/2024/north-america/cfp.html)
```
Comment (natebower) [inline _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md:64]:
```suggestion
[OpenSearchCon 2024: North America CFP](https://opensearch.org/events/opensearchcon/2024/north-america/cfp.html)
```

File: _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+[OpenSearchCon Home Page](https://opensearch.org/events/opensearchcon/2024/north-america/index.html)
```
Comment (natebower) [inline _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md:65]:
```suggestion
[OpenSearchCon 2024: North America](https://opensearch.org/events/opensearchcon/2024/north-america/index.html)
```

**Review by natebower — COMMENTED** [review]
@nateynateynate Please see my comments and changes and let me know if you have any questions. Thanks!

**Comment by natebower** [comment]
> All of the suggested changes look totally fine with me, so from a content perspective I approve. Thanks for the review Nathan, I'll take into account the few style things you fixed multiple times and try not to do that again.

My pleasure 😄

File: _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+
+On September 26, we'll be offering smaller training sessions and workshops with enough room for around 100 people who want to get a more in-depth look at OpenSearch from experts in the field. Separate sign-ups will be provided once the schedule has been finalized, so remember to visit the conference site often for the latest news and session information.
+
+Above all, we just want everyone to come out, have some fun, and learn a lot along the way. See you in San Francisco!
+
+### Resources
+
+* [Registration Page](https://opensearch.org/events/opensearchcon/2024/north-america/register.html)
+* [CFP Submission Page](https://opensearch.org/events/opensearchcon/2024/north-america/cfp.html)
+* [OpenSearchCon Home Page](https://opensearch.org/events/opensearchcon/2024/north-america/index.html)
```
Comment (krisfreedain) [inline _posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md:65]: This should probably be "OpenSearchCon North America Home Page" instead

**Review by krisfreedain — COMMENTED** [review]
tiny little edit recommeneded

**Review by krisfreedain — APPROVED** [review]
Woot!

**Pull request merged**